### DISCUSSION TITLE
Drag and Drop using GH values

### DISCUSSION THREAD

**Discussion by Bayramito** [software-mansion/react-native-gesture-handler#3128] [Q&A] [answered]
Hello,

I just wan't to learn what is the best way to make a drag and drop system using GH values/ 

Let's say you have a basket and an element that you can pan on the screen.
But tricky part is the basket is also moveable. So you can change the basket's position.

I don't want to use native measurement api. 

Is it possible to do something like this using GH ?

**Comment by Glazzes** [comment, marked as answer]
Pan gestures provide absolute values relative to the screen, so you could check whether the current item absolute position is located on top of the absolute position of the basket, of course assuming this is some sort of recycler bin feature.
